### PR TITLE
Allow empty filters

### PR DESCRIPTION
**Pull request by SferaDev**
Closes: #565
Closes: #566

### CODE REVIEW

File: test/integration/query.test.ts [hunk 1]
```diff
@@ -204,6 +204,16 @@ describe('integration tests', () => {
     expect(teams).toHaveLength(0);
   });
 
+  test('filter is empty', async () => {
```
Comment (richardgill) [inline test/integration/query.test.ts:207]: I saw one of the issues was for `.filter({})` should we add a quick test for that too?

**Review by richardgill — APPROVED** [review]
LGTM, added 1 comment

File: test/integration/query.test.ts (hunk 1 above)
Comment (SferaDev) [reply test/integration/query.test.ts:207]: Also included in that test! It's actually kind of empty (we don't send any filters to the API)
